Update 5.0.0-install-instructions.md (#5639)
dotnet/core: release-notes/5.0/5.0.0/5.0.0-install-instructions.md

| Fedora          | dnf     |
| OpenSUSE, SLES  | zypper  |

> Note: Before updating to .NET 5.0, please un-install the preview versions for .NET 5.0.

Reviewer: ```suggestion
Note: Before updating to .NET 5.0, uninstall any preview versions of .NET 5.0 you have previously installed.
```
